easier68k/core/opcodes/bcc.py

```diff
@@ -0,0 +1,737 @@
+# -*- coding: utf-8 -*-
+
+from ...core.enum.condition_status_code import ConditionStatusCode
+from ...simulator.m68k import M68K
+from ...core.opcodes.opcode import Opcode
+from ...core.util import opcode_util
+from ..util.parsing import *
+
+# makes TC way easier lol
```

Comment:
If you have suggestions on how we could improve type conversion overall (perhaps in the base opcode class as well), that would be super useful!

Reply:
TC here refers to Twos Compliment.

Reply:
the Bitstring class will handle signs and casting for me. I find it super convenient.

Reply:
Oh, oops. Yeah 2s comp is used a lot, so maybe we could try to use this instead